dali/test/python/test_pytorch_operator.py

```diff
@@ -40,10 +40,10 @@ def random_seed():
 
 
 class CommonPipeline(Pipeline):
-    def __init__(self, batch_size, num_threads, device_id, _seed, image_dir):
-        super(CommonPipeline, self).__init__(batch_size, num_threads, device_id, seed=_seed,
+    def __init__(self):
+        super(CommonPipeline, self).__init__(BATCH_SIZE, NUM_WORKERS, DEVICE_ID, SEED,
```

Comment:
Why? This makes it impossible to test for varying batch sizes or multiple GPUs. If you don't want to retype, use these as default values instead of deleting the arguments.

Reply:
Done